NightscoutServiceKitTests/OTPManagerTestCase.swift

```diff
@@ -21,77 +21,129 @@ class OTPManagerTestCase: XCTestCase {
         // Put teardown code here. This method is called after the invocation of each test method in the class.
     }
```

Comment:
* Tests should support more than 2 OTPs.
* Added a new test for the case we fail to store used OTPs to disk.
* Miscellaneous test cleanup.